packages/kit/src/core/prerender/prerender.js

```diff
@@ -1,18 +1,34 @@
 import { readFileSync, writeFileSync } from 'fs';
 import { dirname, join } from 'path';
+import sade from 'sade';
 import { pathToFileURL, URL } from 'url';
 import { mkdirp, posixify, walk } from '../../utils/filesystem.js';
 import { installPolyfills } from '../../node/polyfills.js';
 import { is_root_relative, resolve } from '../../utils/url.js';
 import { queue } from './queue.js';
 import { crawl } from './crawl.js';
 import { escape_html_attr } from '../../utils/escape.js';
+import { logger } from '../utils.js';
+import { load_config } from '../config/index.js';
 
 /**
  * @typedef {import('types').PrerenderErrorHandler} PrerenderErrorHandler
  * @typedef {import('types').Logger} Logger
  */
 
+const prog = sade('prerender');
```

Comment:
do we need to use `sade` here? if it's only being used internally we could literally just do

```js
const [, , client_out_dir, results_path, manifest_path, verbose] = process.argv;
```

Reply:
done